controllers/operatorpolicy_status.go

```diff
@@ -1044,7 +1079,7 @@ func noInstallPlansObj(namespace string) policyv1.RelatedObject {
 // existingInstallPlanObj returns a RelatedObject for the InstallPlan, with a reason
 // like 'The InstallPlan is ____' based on the phase. When the InstallPlan is in phase
 // 'Complete', the object will be Compliant, otherwise it will be NonCompliant.
-func existingInstallPlanObj(ip client.Object, phase string) policyv1.RelatedObject {
+func existingInstallPlanObj(ip client.Object, phase string, complianceConfig string) policyv1.RelatedObject {
```

Comment:
I would prefer if this took type `policyv1beta1.ComplianceConfigAction`, there are two conversions to string that would be unnecessary.